gui/wxpython/datacatalog/infomanager.py

```diff
@@ -31,8 +32,10 @@ def __init__(self, infobar, giface):
 
     def ShowDataStructureInfo(self, onCreateLocationHandler):
         """Show info about the data hierarchy focused on the first-time user"""
-        buttons = [("Create new Location", onCreateLocationHandler),
-                   ("Learn More", self._onLearnMore)]
+        buttons = [
+            ("Create new Location", onCreateLocationHandler),
+            ("Learn More", self._onLearnMore),
```

Comment:
1) These two should be translatable.

2) "Learn More" -> "Learn more"

Reply:
Yes, I think that all of the infobar buttons should be translatable...